[bug] grafana-cli invalid memory address

When trying to install a plugin `grafana-cli plugins install worldping-app`

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x565948]

goroutine 1 [running]:
github.com/grafana/grafana/pkg/cmd/grafana-cli/services.ListAllPlugins(0xc82009de60, 0x1f, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/grafana/grafana/pkg/cmd/grafana-cli/services/services.go:20 +0x108
github.com/grafana/grafana/pkg/cmd/grafana-cli/services.GetPlugin(0x7fff2fa0ff6b, 0xd, 0xc82009de60, 0x1f, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/grafana/grafana/pkg/cmd/grafana-cli/services/services.go:70 +0x8c
github.com/grafana/grafana/pkg/cmd/grafana-cli/commands.InstallPlugin(0x7fff2fa0ff6b, 0xd, 0x0, 0x0, 0x7f35f7d04828, 0xc82006e040, 0x0, 0x0)
	/go/src/github.com/grafana/grafana/pkg/cmd/grafana-cli/commands/install_command.go:61 +0xa5
github.com/grafana/grafana/pkg/cmd/grafana-cli/commands.installCommand(0x7f35f7d04828, 0xc82006e040, 0x0, 0x0)
	/go/src/github.com/grafana/grafana/pkg/cmd/grafana-cli/commands/install_command.go:57 +0x25c
github.com/grafana/grafana/pkg/cmd/grafana-cli/commands.runCommand.func1(0xc82008cc60)
	/go/src/github.com/grafana/grafana/pkg/cmd/grafana-cli/commands/commands.go:13 +0xa2
github.com/codegangsta/cli.Command.Run(0x811c50, 0x7, 0x0, 0x0, 0x0, 0x0, 0x0, 0x856ea0, 0x15, 0x0, ...)
	/go/src/github.com/grafana/grafana/Godeps/_workspace/src/github.com/codegangsta/cli/command.go:137 +0x107e
github.com/codegangsta/cli.(*App).RunAsSubcommand(0xc82008ca20, 0xc82008c900, 0x0, 0x0)
	/go/src/github.com/grafana/grafana/Godeps/_workspace/src/github.com/codegangsta/cli/app.go:283 +0x11a0
github.com/codegangsta/cli.Command.startApp(0x8126a8, 0x7, 0x0, 0x0, 0x0, 0x0, 0x0, 0x85d440, 0x1a, 0x0, ...)
	/go/src/github.com/grafana/grafana/Godeps/_workspace/src/github.com/codegangsta/cli/command.go:215 +0x7cc
github.com/codegangsta/cli.Command.Run(0x8126a8, 0x7, 0x0, 0x0, 0x0, 0x0, 0x0, 0x85d440, 0x1a, 0x0, ...)
	/go/src/github.com/grafana/grafana/Godeps/_workspace/src/github.com/codegangsta/cli/command.go:59 +0x197d
github.com/codegangsta/cli.(*App).Run(0xc82008c7e0, 0xc820072000, 0x5, 0x5, 0x0, 0x0)
	/go/src/github.com/grafana/grafana/Godeps/_workspace/src/github.com/codegangsta/cli/app.go:176 +0xfff
main.main()
	/go/src/github.com/grafana/grafana/pkg/cmd/grafana-cli/main.go:55 +0x4d6
```

--- a/pkg/cmd/grafana-cli/services/services.go
+++ b/pkg/cmd/grafana-cli/services/services.go
@@ -15,14 +15,16 @@ var IoHelper m.IoUtil = IoUtilImp{}
 
 func ListAllPlugins(repoUrl string) (m.PluginRepo, error) {
 	fullUrl := repoUrl + "/repo"
-	res, _ := goreq.Request{Uri: fullUrl, MaxRedirects: 3}.Do()
-
+	res, err := goreq.Request{Uri: fullUrl, MaxRedirects: 3}.Do()
+  if err != nil {
+    return m.PluginRepo{}, err
+  }
 	if res.StatusCode != 200 {
 		return m.PluginRepo{}, fmt.Errorf("Could not access %s statuscode %v", fullUrl, res.StatusCode)
 	}
 
 	var resp m.PluginRepo
-	err := res.Body.FromJsonTo(&resp)
+	err = res.Body.FromJsonTo(&resp)
 	if err != nil {
 		return m.PluginRepo{}, errors.New("Could not load plugin data")
 	}

--- a/pkg/cmd/grafana-cli/services/services.go
+++ b/pkg/cmd/grafana-cli/services/services.go
@@ -16,9 +16,9 @@ var IoHelper m.IoUtil = IoUtilImp{}
 func ListAllPlugins(repoUrl string) (m.PluginRepo, error) {
 	fullUrl := repoUrl + "/repo"
 	res, err := goreq.Request{Uri: fullUrl, MaxRedirects: 3}.Do()
-  if err != nil {
-    return m.PluginRepo{}, err
-  }
+	if err != nil {
+		return m.PluginRepo{}, err
+	}
 	if res.StatusCode != 200 {
 		return m.PluginRepo{}, fmt.Errorf("Could not access %s statuscode %v", fullUrl, res.StatusCode)
 	}

